t/redirects.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+describe 'redirecting links' do
+  around { |test| VCR.use_cassette(titles.first, &test) }
+  subject { WikiData.ids_from_pages('en', titles) }
+
+  describe 'redirect and direct link' do
+    let(:titles) { ['The Meaning of Liff', 'The Deeper Meaning of Liff'] }
+
+    it 'pairs both redirecting link and direct link with wikidata ID' do
+      subject.must_equal('The Meaning of Liff' => 'Q875382', 'The Deeper Meaning of Liff' => 'Q875382')
+    end
+  end
+
+  describe 'only redirect links' do
```

Comment:
it's a little hard to follow what specifically this is testing, in part because the simpler case of a single item that is a redirect doesn't have its own test. Is this meant to _be_ that case? (and if so, why test two?) Or is this testing something more than that? (in which case, it would probably be better to have the simpler case separately, and make it more obvious what this one is testing)